batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/exception/ValidatorException.java

```diff
@@ -0,0 +1,20 @@
+package com.lftechnology.batch7crud.exception;
+
+import java.util.Map;
+
+/**
+ * Created by Prajjwal Raj Kandel <prajjwalkandel@lftechnology.com> on 1/25/16.
+ */
+public class ValidatorException extends Exception {
+  private Map<String,String> errors;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "errors" field final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1165)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make "errors" transient or serializable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1948)